eng/vcpkg-commit.txt

```diff
@@ -1 +1 @@
-fcd3a3bb53356cc954f7fa07cee2c99b253d1a49
+be5c4ef68b51142ba705f0678b45d284977de677
```

Comment:
Why did we change this?

Reply:
To fix CI failure as suggested by @danieljurek . It worked

Reply:
Looks like we picked the recent commit from vcpkg:
https://github.com/microsoft/vcpkg/commit/be5c4ef68b51142ba705f0678b45d284977de677

![image](https://user-images.githubusercontent.com/6527137/157351758-07ff284c-d278-4128-b446-2022b617852b.png)

I am not sure if it is the best idea to point to any arbitrary commit from vcpkg, but maybe that's good enough. We need to consider adding some mechanism to test the SDK, against the oldest dependencies we support, to validate things continue to work for customers.

The last time we bumped it, it was a commit from Nov.
https://github.com/Azure/azure-sdk-for-cpp/pull/3154

**That said, picking a sha right after the latest azure core release makes sense to me!**
In this case, that would be:
97113b43b3b6905a8c583900a0540471cfa88d6e
https://github.com/microsoft/vcpkg/pull/23371

Reply:
This is the testing strategy vcpkg folks suggest:
https://github.com/Azure/azure-sdk-for-cpp/pull/3154#discussion_r763468105